sdk/keyvault/azure-security-keyvault-keys/src/list_keys_single_page_result.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/keyvault/keys/list_keys_single_page_result.hpp"
+#include "azure/keyvault/keys/details/key_constants.hpp"
+
+#include <azure/keyvault/common/internal/unix_time_helper.hpp>
+
+#include <azure/core/internal/json/json.hpp>
+#include <azure/core/internal/json/json_optional.hpp>
+#include <azure/core/internal/json/json_serializable.hpp>
+#include <azure/core/url.hpp>
+
+using namespace Azure::Security::KeyVault::Keys;
+using namespace Azure::Core::Json::_internal;
+using Azure::Security::KeyVault::Common::_internal::UnixTimeConverter;
+
+KeyPropertiesSinglePage _detail::KeyPropertiesSinglePageDeserialize(
+    Azure::Core::Http::RawResponse const& rawResponse)
+{
+  KeyPropertiesSinglePage result;
+  auto& body = rawResponse.GetBody();
+  auto jsonParser = json::parse(body);
+
+  JsonOptional::SetIfExists(result.ContinuationToken, jsonParser, "nextLink");
```

Comment:
There are lots of hardcoded key strings throughout the code.  We should keep these all in one place as constants

Reply:
I tried to hardcode only when they are used only once, otherwise, I would add it to constants.  I will check this case and others.